File: MODUL-1/SOAL 8/PRAK108-2410817120017-RIKA-FAULIANA-RAHMI.c
#include <stdio.h>

int main() {
    // Diketahui
    int putaran = 5;  // jumlah putaran
    float jarak_tempuh = 14.0;  // jarak tempuh dalam kilometer

    // Menghitung keliling taman
    float keliling_taman = jarak_tempuh / putaran;  // keliling taman dalam kilometer

    // Menghitung jari-jari taman menggunakan rumus keliling = 2 * pi * r
    float pi = 3.14159265359;  // nilai pi
    float jari_jari = keliling_taman / (2 * pi);  // jari-jari taman dalam kilometer

    // Menampilkan hasil
    printf("Diketahui : \n");
    printf("Pak Dengklek mengelilingi taman = %d Putaran \n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.2f Kilometer \n\n", jarak_tempuh);
    printf("Jawaban : \n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jari_jari);

    return 0;
}

File: MODUL-1/SOAL 8/PRAK108-2410817120017-RIKA-FAULIANA-RAHMI.py
import math

# Diketahui
putaran = 5
jarak_tempuh = 14  # dalam kilometer

# Menghitung keliling satu putaran
keliling_satu_putaran = jarak_tempuh / putaran

# Menghitung jari-jari taman
jari_jari = keliling_satu_putaran / (2 * math.pi)

# Output
print("Diketahui :")
print(f"Pak Dengklek mengelilingi taman = {putaran} Putaran")
print(f"Jarak tempuh Pak Dengklek = {jarak_tempuh} Kilometer\n")

print("Jawaban :")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jari_jari:.2f} Kilometer")
